src/components/NavSection.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <div class="Home">
+    <sidebar-section v-if="showSidebar" />
+    <nav>
+      <div class="logo">
+        <img src="@/assets/images/Icons/Group 2.svg" alt="" id="logo" />
+      </div>
+      <div class="nav-options">
+        <div class="nav-icon">
+          <img src="@/assets/images/Icons/Rectangle.svg" alt="" id="nav-icon" />
+        </div>
+        <ul>
+          <li> <router-link to ="/"><span>00</span>HOME</router-link></li>
+          <li><span>01</span>DESTINATION</li>
+          <li><span>02</span>CREW</li>
+          <li><span>03</span>TECHNOLOGY</li>
+        </ul>
+      </div>
+      <div class="nav-icons" @click="toggleIcon">
+        <img
+          src="@/assets/images/Icons/Group (2).svg"
+          alt=""
+          id="hamburger-icon"
+          v-if="showIcon"
+        />
+        <img src="@/assets/images/Icons/Group (1).svg" alt="" id="close-icon" v-else />
+      </div>
+    </nav>
+  </div>
+</template>
+<script>
+import SidebarSection from "@/components/SidebarSection.vue";
+export default {
+  components: {
+    SidebarSection,
+  },
+  data() {
+    return {
+      showIcon: true,
+      showSidebar: false,
+    };
+  },
+  methods: {
+    toggleIcon() {
+      this.showIcon = !this.showIcon;
```

Comment:
You don't need this additional property

You already have `showSidebar`
Check my comment on line 24